app/account_linking.py

```diff
@@ -168,6 +168,8 @@ def process(self) -> LinkResult:
 
 class ExistingUnlinkedUserStrategy(ClientMergeStrategy):
     def process(self) -> LinkResult:
+        # IF it was scheduled to be deleted. Unschedule it.
+        self.user.delete_on = None
```

Comment:
I'm not familar with this code part, maybe you can ask Carlos to have a look.

Reply:
Yep. Already discussed it with him.